### DISCUSSION TITLE
preview issue in minio

### DISCUSSION THREAD

**Discussion by tharung07** [minio/minio#19426] [Q&A] [answered]
Hi Everyone,

I am unable to preview ms office file in minio. Could you please provide me solution ?
![Screenshot 2024-04-06 154807](https://github.com/minio/minio/assets/165164157/77214b44-2b71-409a-b56a-45015a3ca137)

**Comment by harshavardhana** [comment, marked as answer]
No there is no preview for such files. We only preview what is supported by html5 browser support.
